packages/webapp/src/Routes.js

```diff
@@ -135,6 +138,7 @@ const Routes = () => {
         <Switch>
           <Route path="/" exact component={Home} />
           <Route path="/home" exact component={Home} />
+          <Route path="/custom-sign-up" exact component={CustomSignUp} />
```

Comment:
I removed the one on line 237 (which shouldn't be there), but the route is added to line 141 because it is part of the onboarding flow. If removed, nothing will be rendered when a user tries to create a new account.

Reply:
Why should a user create a new account when logged in?

Reply:
A user would not even be able to arrive at this route is they are not logged in -- that's why this route is placed under if `(!isAuthenticated)`